### PR TITLE
Update dark-sites.config

### PR DESCRIPTION
**Pull request by sprinkledcheese**
Added humblebundle, humblegames. Removed bing since it's not dark and basho because it's inaccesible.

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -36,7 +35,6 @@ beta.jdownloader.org
 betterttv.com
 bherila.net
 bin.disroot.org
-bing.com/$
```
Comment (Gusted) [inline src/config/dark-sites.config:39]: Home-page

Comment (Gusted) [reply src/config/dark-sites.config:39]: Is broken with darkreader on

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -169,6 +167,12 @@ heavybit.com
 hidive.com
 himovies.to
 hostedtalk.net
+humblebundle.com/$
```
Comment (Gusted) [inline src/config/dark-sites.config:170]: Why doing this well already refericing to other sub-url's?

File: src/config/dark-sites.config (hunk 1 above)
Comment (sprinkledcheese) [reply src/config/dark-sites.config:39]: ok.

File: src/config/dark-sites.config (hunk 2 above)
Comment (sprinkledcheese) [reply src/config/dark-sites.config:170]: Because those other sub-url's have dark mode.

**Pull request merged**